daemon/volumes.go

```diff
@@ -103,6 +103,9 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		for _, m := range c.MountPoints {
+			if _, ok := mountPoints[m.Destination]; ok {
+				return fmt.Errorf("Duplicate mount point '%s'", m.Destination)
```

Comment:
@cpuguy83 @thaJeztah Got that, thanks. Was wondering how to best handle this situation. Will take a look and make sure the previous behavior is preserved.